ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/operations/AttestationData.java

```diff
@@ -74,6 +75,12 @@ public UInt64 getEarliestSlotForForkChoice(final Spec spec) {
     return getSlot().plus(UInt64.ONE).max(getTarget().getEpochStartSlot(spec));
   }
 
+  public UInt64 getEarliestSlotForForkChoice(final MiscHelpers miscHelpers) {
```

Comment:
there is a bit of duplication, but at the end is ok.

Reply:
It's just that in `AttestationUtilPhase0` there is no access to `Spec`, so just using the miscHelpers instead.